release-tools/release-cc.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+script_name="${0##*/}"
+script_dir="$(dirname $(realpath -s "$0"))"
+release_tool="${script_dir}/release-tool"
+owner=${OWNER:-clearcontainers}
+
+function usage() {
+cat << EOT
+Usage: ${script_name}
+This script creates a new release for Clear Containers
+It tags and create release for:
+Proxy
+Shim
+Runtime
+
+Enviroment Variables:
+
+GITHUB_TOKEN: Export GITHUB_TOKEN variable with a valid token and repository permissions
+       See: https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/
+EOT
+}
+
+die()
+{
+	msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+function build_release_tool() {
+go build -o  "${release_tool}"
+}
+
+function check_token() {
+if [ -z "$GITHUB_TOKEN" ]; then
+	echo "token is empty"
+	usage
+	exit
+fi
+}
+
+function release(){
+commit="master"
+
+expected_next_version="$(${release_tool} status --next-bump runtime)"
+runtime_version=$(curl -s https://raw.githubusercontent.com/clearcontainers/runtime/${commit}/VERSION)
```

Comment:
- Could you add a variable for `https://raw.githubusercontent.com/clearcontainers/`?
- It might be safer to change all the `curl` lines to be `curl -Ls`.